[32m2021-11-11 22:12:18.936[0m | [1mINFO    [0m | [36mhttprunner.runner[0m:[36mtest_start[0m:[36m450[0m - [1mStart to run testcase: case003, TestCase ID: 27d12edc-eadb-4905-a7f3-f4a39a03b7c8[0m
[32m2021-11-11 22:12:18.937[0m | [1mINFO    [0m | [36mhttprunner.runner[0m:[36m__run_step[0m:[36m292[0m - [1mrun step begin: step_name >>>>>>[0m
[32m2021-11-11 22:12:22.962[0m | [33m[1mWARNING [0m | [36mhttprunner.client[0m:[36mrequest[0m:[36m188[0m - [33m[1mfailed to get client address info: 'NoneType' object has no attribute 'connection'[0m
[32m2021-11-11 22:12:22.962[0m | [33m[1mWARNING [0m | [36mhttprunner.client[0m:[36mrequest[0m:[36m196[0m - [33m[1mfailed to get server address info: 'NoneType' object has no attribute 'connection'[0m
[32m2021-11-11 22:12:22.963[0m | [34m[1mDEBUG   [0m | [36mhttprunner.client[0m:[36mlog_print[0m:[36m40[0m - [34m[1m
================== request details ==================
method   : GET
url      : http://localhost/demo/path
headers  : {}
cookies  : {}
body     : None
[0m
[32m2021-11-11 22:12:22.968[0m | [34m[1mDEBUG   [0m | [36mhttprunner.client[0m:[36mlog_print[0m:[36m40[0m - [34m[1m
================== response details ==================
status_code : 0
headers  : {}
cookies  : {}
encoding : None
content_type : 
body     : 
[0m
[32m2021-11-11 22:12:22.968[0m | [31m[1mERROR   [0m | [36mhttprunner.client[0m:[36mrequest[0m:[36m215[0m - [31m[1mHTTPConnectionPool(host='localhost', port=80): Max retries exceeded with url: /demo/path (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001157F04EF40>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))[0m
[32m2021-11-11 22:12:22.969[0m | [1mINFO    [0m | [36mhttprunner.runner[0m:[36m__run_step[0m:[36m304[0m - [1mrun step end: step_name <<<<<<
[0m
[32m2021-11-11 22:12:22.970[0m | [1mINFO    [0m | [36mhttprunner.runner[0m:[36mtest_start[0m:[36m460[0m - [1mgenerate testcase log: D:\Course\course_django\autotpsite\logs\27d12edc-eadb-4905-a7f3-f4a39a03b7c8.run.log[0m
